File: app/src/main/java/androidstudio/class27/java/ui/AdapterStudentListViewHolder.java
package androidstudio.class27.java.ui;

import android.view.View;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

import com.rafaelmfer.codeclasses.R;

class AdapterStudentListViewHolder extends RecyclerView.ViewHolder {

    TextView tvStudentName, tvStudentRa;

    AdapterStudentListViewHolder(@NonNull View itemView) {
        super(itemView);
        tvStudentName = itemView.findViewById(R.id.tvStudentName);
        tvStudentRa = itemView.findViewById(R.id.tvStudentRa);
    }
}

File: app/src/main/java/androidstudio/class27/kotlin/ui/AdapterStudentListViewHolder.kt
package androidstudio.class27.kotlin.ui

import android.view.View
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.rafaelmfer.codeclasses.R

class AdapterStudentListViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    var tvStudentName: TextView = itemView.findViewById(R.id.tvStudentName)
    var tvStudentRa: TextView = itemView.findViewById(R.id.tvStudentRa)
}